LuaMenu/widgets/gui_color_change_window.lua

```diff
@@ -0,0 +1,163 @@
+	{{ 1.00, 0.84, 0.71 }, "Apricot"},
+	{{ 0.00, 0.00, 0.50 }, "Navy"},
+	{{ 0.50, 0.50, 0.50 }, "Grey"},
+	{{ 1.00, 1.00, 1.00 }, "White"},
+	{{ 0.00, 0.00, 0.00 }, "Black"}
+}
+
+local colorMap = {}
+local colorNames = { '...' }
+for _, color in ipairs(colors) do
+	colorMap[color[2]] = color[1]
+	table.insert(colorNames, color[2])
+end
+
+local function CreateColorChangeWindow(opts)
+	opts = opts or {}
+	local selectedColor = opts.initialColor
+	local OnAccepted = opts.OnAccepted or function() end
+
+	local Configuration = WG.Chobby.Configuration
+
+	local colorChangeWindow = Window:New {
+		caption = "",
+		name = "colorChangeWindow",
+		parent = screen0,
+		width = 280,
+		height = 330,
+		resizable = false,
+		draggable = false,
+		classname = "main_window",
+	}
+
+	local ChangeAccepted = function()
+		OnAccepted(selectedColor)
+	end
+	local CloseFunction = function()
+		colorChangeWindow:Dispose()
+		colorChangeWindow = nil
+	end
+
+	local lblTitle = Label:New {
+		x = 0,
+		y = 15,
+		width = colorChangeWindow.width - colorChangeWindow.padding[1] - colorChangeWindow.padding[3],
+		height = 35,
+		align = "center",
+		font = Configuration:GetFont(4),
+		caption = "Choose color",
+		parent = colorChangeWindow,
+	}
+
+	local btnOK = Button:New {
+		x = "10%",
+		width = "30%",
+		bottom = 1,
+		height = 40,
+		caption = i18n("ok"),
+		font = Configuration:GetFont(2),
+		classname = "action_button",
+		OnClick = { CloseFunction, ChangeAccepted },
+		parent = colorChangeWindow,
+	}
+
+	local btnCancel = Button:New {
+		right = "10%",
+		width = "30%",
+		bottom = 1,
+		height = 40,
+		caption = i18n("cancel"),
+		font = Configuration:GetFont(2),
+		classname = "action_button",
+		OnClick = { CloseFunction },
+		parent = colorChangeWindow,
+	}
+
+	local imTeamColor
+	local cmbColors = ComboBox:New {
+		x = colorChangeWindow.width * 0.05,
+		y = 150,
+		width = colorChangeWindow.width * 0.5,
+		height = 40,
+		items = colorNames,
+		parent = colorChangeWindow,
+		font = Configuration:GetFont(2),
+		itemFontSize = Configuration:GetFont(2).size,
+		itemHeight = 30,
+		selectByName = true,
+		OnSelectName = {
+			function (obj, name)
+				local color = colorMap[name]
+				if color == nil then
+					return
+				end
+				selectedColor = color
+				imTeamColor.color = color
+				imTeamColor:Invalidate()
+			end
+		},
+	}
+
+	imTeamColor = Image:New {
+		name = "imTeamColor",
+		x = cmbColors.x + cmbColors.width + 20,
+		y = 150,
+		width = 40,
+		height = 40,
+		parent = colorChangeWindow,
+		keepAspect = false,
+		file = "LuaMenu/widgets/chili/skins/Evolved/glassBk.png",
+		color = selectedColor,
+	}
+
+	WG.Chobby.PriorityPopup(colorChangeWindow, CloseFunction, CloseFunction, screen0)
+end
+
+
+function DelayedInitialize()
+	local lobby = WG.LibLobby.lobby
+end
+
+
+function widget:Initialize()
+	VFS.Include(LUA_DIRNAME .. "widgets/chobby/headers/exports.lua", nil, VFS.RAW_FIRST)
+
+	WG.Delay(DelayedInitialize, 1)
+end
+
+WG.ColorChangeWindow = {
```

Comment:
Initialising into WG is unadvised during file reading. The things this depends on could not exist.